exporter/otlpexporter/README.md

```diff
@@ -28,6 +28,16 @@ The following settings can be optionally configured:
   [grpc.WithInsecure()](https://godoc.org/google.golang.org/grpc#WithInsecure).
 - `balancer_name`(default = pick_first): Sets the balancer in grpclb_policy to discover the servers.
 See [grpc loadbalancing example](https://github.com/grpc/grpc-go/blob/master/examples/features/load_balancing/README.md).
+- `timeout` (default = 5s): Is the timeout for each operation.
+- `retry_settings`
+  - `disabled` (default = false)
```

Comment:
Do we have a `enabled`/`disabled` flag already elsewhere in the project? If not, it would be best to have `enabled` instead of `disabled`: having something happening on double negatives is confusing to users (`disabled: false` == enabled)

Reply:
Since this is not going to be released for at least 1 week, I decided to create an issue https://github.com/open-telemetry/opentelemetry-collector/issues/1397.

Right now we don't use any of them, so a good moment to decide on one or the other.